TASK-3/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using TASK;
+
+var persons = new List<Person>();
+persons.Add(new Person("Max", "Smith", new DateOnly(2000, 06, 05), true));
+persons.Add(new Person("Liz", "Mueller", DateOnly.Parse("1970, 07, 21"), true));
+persons.Add(new Person("Meghan", "Markle", DateOnly.Parse("1999, 02, 18"), true));
+
+var drivingLicenses = new List<DrivingLicense>();
+drivingLicenses.Add(new DrivingLicense(DateOnly.Parse("2020, 01, 01"), Guid.NewGuid()));
+drivingLicenses.Add(new DrivingLicense(DateOnly.Parse("1990, 03, 12"), Guid.NewGuid()));
+drivingLicenses.Add(new DrivingLicense(DateOnly.Parse("2017, 08, 11"), Guid.NewGuid()));
+
+var engines = new List<Engine>();
+engines.Add(new Engine(15, 30, 6, "Oil", 300));
+engines.Add(new Engine(4, 70, 10, "Diesel", 120));
+engines.Add(new Engine(3, 10, 5, "Gasoline", 80));
+
+var vehicles = new List<Vehicle>();
+vehicles.Add(new Vehicle("BMW", 2005, engines[0]));
+vehicles.Add(new Vehicle("VW", 1981, engines[1]));
+vehicles.Add(new Vehicle("Pegeout", 1970, engines[2]));
+
+var drivers = new List<Driver>();
+for (var i = 0; i < persons.Count; i++)
+{
+    drivers.Add(new Driver(persons[i], drivingLicenses[i], vehicles[i]));
+}
+
+Console.WriteLine(" №  | {0,30} | {1,15} | {2,30} | {3}", "Driver Name:", "Date of Birth", "Date of Driving License:",
+    "Id Number");
+
+for (var i = 0; i < drivers.Count; i++)
+{
+    Console.WriteLine(" {4}. | {0,30} | {1,15} | {2,30} | {3} ",
+        drivers[i].Person.LastName + " " + drivers[i].Person.FirstName, drivers[i].Person.DateOfBirth,
+        drivers[i].DrivingLicense.DateDrivingLicense,
+        drivers[i].DrivingLicense.IdNumber, i + 1);
+}
+
+
+var driverChoice = 0;
+do
+{
+    Console.Write("Choose the driver (1-" + drivers.Count + "): ");
+    driverChoice = Convert.ToInt32(Console.ReadLine());
+} while (driverChoice < 0 || driverChoice > drivers.Count);
```

Comment:
A person can enter 0 in your program and continue running the program, fix it

Reply:
Changed driverChoice < 0 to driverChoice <= 0.